static/src/stylesheets/layout/new-header/_menu.scss

```diff
@@ -35,8 +38,27 @@
         margin-right: $gutter-xlarge + ($veggie-burger-medium / 2);
     }
 
+    // based on https://css-tricks.com/full-width-containers-limited-width-parents/#article-header-id-6
+    @include mq(desktop) {
+        display: none;
+        left: 50%;
+        margin-left: -50vw;
```

Comment:
Haven't done any cross-browser testing so far. I'd say that we should start doing this, once we have the rough layout in place, so we can develop fast now and towards the end start fixing bugs ...